lib/url-template.js

```diff
@@ -13,16 +13,36 @@
   function UrlTemplate() {
   }
 
+  var percentTwenties = /%20/g;
+
+  /**
+   * @private
+   * @param {string} str
```

Comment:
Nit: do we need the `str` here?

Reply:
tbh unsure, I was just following the formatting in the other methods, so it may not be necessary at all